features/providers/civil_application_journey.feature

```diff
@@ -153,6 +153,10 @@ Feature: Civil application journeys
     Then I should be on a page showing 'Harassment - injunction'
     Then I should be on a page showing 'Emergency certificate'
     Then I should be on a page showing 'Substantive certificate'
+    Then I should be on a page showing "Do you want to request a higher cost limit?"
```

Comment:
Should this be testing if the Setting to allow multiple proceedings is disabled? 

because the emergency_cost_override.feature tests when it's enabled

Reply:
my reading was that this feature goes live with multiple proceedings. I will clarify with Dave/Simon because right now this feature will not be visible when multi proc is set to off

Reply:
Confirmed this is the correct behaviour, new feature will go live with MP